app/assets/javascripts/host_edit.js

```diff
@@ -431,7 +431,6 @@ function override_param(item){
   var new_param = $('#parameters').find('.fields').last();
   new_param.find('[id$=_name]').val(n);
   new_param.find('[id$=_value]').val(v == param_value.data('hidden-value') ? '' : v);
-  mark_params_override();
```

Comment:
we already know which parameter is overridden, no need to go over all parameters and check again.